<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myfirstgui</class>
 <widget class="QDialog" name="myfirstgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My First Gui!</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-300</x>
     <y>260</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QDial" name="Vatn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Number_Vodka">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
  </widget>
  <widget class="QDial" name="Saft">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="Vodka">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>MIX</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Number_Vatn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Number_Saft">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_Vodka">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Vodka</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Vatn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Vatn</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Saft">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Saft</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>myfirstgui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>myfirstgui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
